STAR/draft-ietf-acme-star.md

```diff
@@ -462,37 +466,39 @@ The Server SHOULD include the "Not-Before" and "Not-After" HTTP headers in the r
 Their purpose is to enable client implementations that do not parse the certificate.
 
-To improve robustness, the next certificate MUST be made available by the ACME CA at the latest halfway through the lifetime of the currently active certificate.
+To improve robustness, the next certificate MUST be made available by the ACME CA at the URL pointed by "star-certificate" at the latest halfway through the lifetime of the currently active certificate.
 It is worth noting that this has an implication in case of cancellation: in fact, from the time the next certificate is made available, the cancellation is not completely effective until the latter also expires.
+To avoid the client accidentally entering a broken state, the "next" certificate MUST be pre-dated so that it is already valid when it is published at the "star-certificate" URL.  For further discussion on pre-dating, see {{operational-cons-clocks}}.
 
 The server MUST NOT issue any additional certificates for this order beyond its recurrent-end-date.
 
-Immediately after the order expires, the server MUST respond with 403 (Forbidden) to any requests to the certificate endpoint.  The response SHOULD provide additional information using a problem document {{RFC7807}} with type "urn:ietf:params:acme:error:recurrentOrderExpired".
+Immediately after the order expires, the server MUST respond with 403 (Forbidden) to any requests to the star-certificate endpoint.  The response SHOULD provide additional information using a problem document {{RFC7807}} with type "urn:ietf:params:acme:error:recurrentOrderExpired".
 
 ## Negotiate unauthenticated GET
 {: #certificate-get-nego }
 
 In order to enable the name delegation workflow defined in
 {{I-D.sheffer-acme-star-request}} as well as to increase the reliability of the
 STAR ecosystem (see {{dependability}} for details), this document defines a
-mechanism that allows a server to advertise support for accessing certificate
-resources via unauthenticated GET (instead of, or in addition to, POST-as-GET),
-and a client to enable this service with per-Order granularity.
+mechanism that allows a server to advertise support for accessing
+star-certificate resources via unauthenticated GET (instead of, or in addition
+to, POST-as-GET), and a client to enable this service with per-Order
+granularity.
 
 Specifically, a server states its availability to grant unauthenticated access
-to a client's Order certificate(s) by setting the star-allow-certificate-get
+to a client's Order star-certificate by setting the star-allow-certificate-get
 key to true in the meta field of the Directory object:
 
 - star-allow-certificate-get (optional, boolean): If this field is present and
-  set to true, the server allows GET requests to certificate URLs.
+  set to true, the server allows GET requests to star-certificate URLs.
```

Comment:
Needs IANA registration, and similarly the per-Order flag and the "star-certificate" field.

Reply:
good catch, thanks.